src/wms/src/main/java/org/geoserver/wms/map/GetMapKvpRequestReader.java

```diff
@@ -96,16 +108,44 @@
     /** EntityResolver provider, used in SLD parsing */
     EntityResolverProvider entityResolverProvider;
 
+    /** HTTP client used to fetch remote styles * */
+    CloseableHttpClient httpClient;
```

Comment:
I don't see attempts at closing it though (if it's kept as a field, should be done when the spring app-context goes down, making the bean implement DisposableBean)